public-ui/src/environments/environment.local.ts

```diff
@@ -0,0 +1,10 @@
+import {testEnvironmentBase} from 'environments/test-env-base';
+
+export const environment = {
+  displayTag: 'Local->Local',
+  allOfUsApiUrl: 'http://localhost:8081',
```

Comment:
Do you need the `allOfUsApiUrl`? Or just the public api url?

Reply:
Removed. One thing i will need is the workbench url for sending people there. Not sure where to get that yet or if i should just put  make an environment variable.